source/nodejs/adaptivecards-controls/src/adaptivecards-controls.css

```diff
@@ -1,363 +1,450 @@
 .ms-ctrl-overlay {
-  position: absolute;
-  left: 0;
-  top: 0; }
+    position: absolute;
+    left: 0;
+    top: 0;
+    z-index: 10000;
+}
 
 @keyframes fadeIn {
-  0% {
-    opacity: 0; } }
+    0% {
+        opacity: 0;
+    }
+}
 
 @keyframes slideTopToBottom {
-  0% {
-    transform: translate(0, -20px); } }
```

Comment:
only spacing changes here looks like right?

Reply:
I am not seeing this in my local file. What I have is correct. Don't know why lines 11 and 12 are marked as removed, they shouldn't be:
![image](https://user-images.githubusercontent.com/1334689/60213618-bbdd7280-9818-11e9-95e7-3b1c75e3a9dc.png)

Reply:
Thanks good to know -- let me check. whitespace changes here.. formatting looks like? 

---
In reply to: [297859608](https://github.com/microsoft/AdaptiveCards/pull/2995#discussion_r297859608) [](ancestors = 297859608)

Reply:
Maybe.